docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
     restart: always
     logging: *default-logging
   resource:
-    image: semtech/mu-cl-resources:1.20.0
+    image: semtech/mu-cl-resources:latest
```

Comment:
Why not use a fixed version ? I'm always a bit scared of `latest`, we don't really have control of what goes there.
Plus I just tested and with latest mu-cl-resource breaks on my stack, but v1.20.0 seems to be working as expected